docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1244,7 +1244,7 @@ Alias: `format`
 [[formatnumber_fn]]
 === `formatnumber`
 
-Formats a number into a formatted number string using NumeralJS. For more information, see http://numeraljs.com/#format.
+Formats a number into a formatted number string using the <<numeral, Elastic Numeral>> pattern syntax.
```

Comment:
Use Numeral instead of Elastic Numeral.  We typically reserve reserve Elastic for use in solution names.

Changes are also required elsewhere in the doc.